vyxal/LazyList.py

```diff
@@ -142,11 +177,12 @@ def filter(self, fn):
                 yield item
 
     def listify(self):
-        from vyxal.helpers import vyxalify
-
-        temp = self.generated + vyxalify(list(self.raw_object))
-        self.raw_object = iter(temp[::])
-        self.generated = []
+        temp = self.generated[::]
+        while True:
+            try:
+                temp.append(self.__next__())
```

Comment:
don't forget to call `vyxalify` here - some generators might return tuples.

Reply:
```suggestion
                temp.append(vyxalify(self.__next__()))
```

Reply:
Wait __next__ itself should call vyxalify

Reply:
Yeah next does call vyxalify, it's all god